packages/app/features/Widget/components/AptosTokensTab/DirectTransfer.tsx

```diff
@@ -18,21 +18,30 @@ const indicatorSize = 16;
 const DirectTransfer: FC<Props> = ({ pubkey, directTransfer, fee }) => {
 	const [disabled, setDisabled] = useState(false);
 
+	const handleOnPasscodeComplete = async (passcode: string) => {
+		const payload: aptosHandler.AptosDirectTransferPayload = {
+			pubkey,
+			directTransfer: !directTransfer,
+		};
+
+		const res = await utils.handleAptosFunction({
+			passcode,
+			type: RequestType.UPDATE_DIRECT_TRANSFER_ON_APTOS,
+			payload,
+			callback: (privateKey, payload) =>
```

Comment:
I dont think we need this payload, choosing corresponding handle must be in kernel (handle by above type), also `@walless/kernel` just used by apps (mobile, web), not in `@walless/app`